### ISSUE TITLE
Assertion failed: (strcmp(SQLITE_VERSION, sqlite3_libversion()) == 0)

### ISSUE THREAD

**Issue by sandacn** [NASA-SW-VnV/homebrew-core#3]
==> cmake -G Unix Makefiles -DGMP_ROOT=/usr/local/opt/gmp -DMPFR_ROOT=/usr/local/opt/mpfr -DPPL_ROOT=/usr/local/opt/ppl -DAPRON_ROOT=/usr/local/opt/apron
Last 15 lines from /Users/ryan/Library/Logs/Homebrew/ikos/04.cmake:
CMake Error at cmake/FindSQLite3.cmake:88 (message):
  error when running a program linked with SQLite3:

  Assertion failed: (strcmp(SQLITE_VERSION, sqlite3_libversion()) == 0),
  function main, file
  /tmp/ikos-20200806-6318-125c8n1/ikos-3.0/build/analyzer/FindSQLite3Version.c,
  line 8.

  Child aborted
Call Stack (most recent call first):
  analyzer/CMakeLists.txt:102 (find_package)

-- Configuring incomplete, errors occurred!
See also "/tmp/ikos-20200806-6318-125c8n1/ikos-3.0/build/CMakeFiles/CMakeOutput.log".

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/nasa-sw-vnv/homebrew-core/issues

**Comment by arthaud** [comment]
Hi @sandacn,

This assertion checks that the SQLite header file matches with the library file. In your case, it doesn't match. Please check whether you have multiple version of SQLite installed.

You can try to compile the following C file with `cc -v` and check the location of the SQLite header file and SQLite library:
```
#include <assert.h>
#include <stdio.h>
#include <string.h>
#include <sqlite3.h>
int main() {
  assert(strcmp(SQLITE_VERSION, sqlite3_libversion()) == 0);
  printf(\"%s\", sqlite3_libversion());
  return 0;
 }
```

**Comment by sandacn** [comment]
```bash
cc -v
```
result: 
```
Apple clang version 11.0.3 (clang-1103.0.32.62)
Target: x86_64-apple-darwin19.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```
compile the c file you given and execute it 
```bash
cc t.c -o t -lsqlite3
./t
```
result： 
```
3.28.0
```

**Comment by sandacn** [comment]
when i reinstall ikos, new issue occurs:

==> python3 -s /private/tmp/ikos--homebrew-virtualenv-20200806-18794-1yi5vcx/target/bin/virtualenv -p python3 /usr/local/Cellar/ikos/3.0/libexec/vendor
Last 15 lines from /Users/ryan/Library/Logs/Homebrew/ikos/02.python3:
Traceback (most recent call last):
  File "/private/tmp/ikos--homebrew-virtualenv-20200806-18794-1yi5vcx/target/lib/python3.8/site-packages/virtualenv.py", line 2632, in <module>
    main()
  File "/private/tmp/ikos--homebrew-virtualenv-20200806-18794-1yi5vcx/target/lib/python3.8/site-packages/virtualenv.py", line 860, in main
    create_environment(
  File "/private/tmp/ikos--homebrew-virtualenv-20200806-18794-1yi5vcx/target/lib/python3.8/site-packages/virtualenv.py", line 1173, in create_environment
    install_wheel(to_install, py_executable, search_dirs, download=download)
  File "/private/tmp/ikos--homebrew-virtualenv-20200806-18794-1yi5vcx/target/lib/python3.8/site-packages/virtualenv.py", line 1019, in install_wheel
    _install_wheel_with_search_dir(download, project_names, py_executable, search_dirs)
  File "/private/tmp/ikos--homebrew-virtualenv-20200806-18794-1yi5vcx/target/lib/python3.8/site-packages/virtualenv.py", line 1110, in _install_wheel_with_search_dir
    call_subprocess(cmd, show_stdout=False, extra_env=env, stdin=script)
  File "/private/tmp/ikos--homebrew-virtualenv-20200806-18794-1yi5vcx/target/lib/python3.8/site-packages/virtualenv.py", line 963, in call_subprocess
    raise OSError("Command {} failed with error code {}".format(cmd_desc, proc.returncode))
OSError: Command /usr/local/Cellar/ik...vendor/bin/python3.8 - setuptools pip wheel failed with error code 1
Running virtualenv with interpreter /usr/local/opt/python@3.8/bin/python3

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/nasa-sw-vnv/homebrew-core/issues

**Comment by ivanperez-keera** [comment]
We've just announced a release candidate for IKOS 3.2, which includes a fix for this issue. We've included instructions to install it with homebrew. See https://github.com/NASA-SW-VnV/ikos/discussions/253 for details.

Please try it and let us know if the issue persists. Thanks!

**Comment by sandacn** [comment]
close issue

**Comment by ivanperez-keera** [comment]
Ok. We'll wait until the formula is merged and then close all pending issues.

**Comment by ivanperez-keera** [comment]
Closed by the solution to #9.
